### PR TITLE
ticdc: update some usage restrictions and configs

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

ticdc: update some usage restrictions and configs

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4474
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -61,7 +61,8 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235
 
 > **Note:**
 >
-> TiCDC (the 4.0 version) does not support dynamically modifying the file sorting and memory sorting yet.
+> + TiCDC (the 4.0 version) does not support dynamically modifying the file sorting and memory sorting yet.
+> + Currently, the file sorting feature only has limited processing capacity. If too large a  data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:65]:
```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If accumulating data of a single table is too large and causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```

Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:65]: What about the following version?
```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If the data accumulated on a single table is too large and causes the file sorting to fail, you can modify the task configuration of TiCDC to filter this table and use other backup and restore tools such as [BR](/br/backup-and-restore-tool.md) to restore the table before you resume replicating the table.
```

Comment (ran-huang) [inline ticdc/troubleshoot-ticdc.md:65]:
```suggestion
> + Currently, the file sorting feature only has limited processing capacity. If too large a data volume on a single table causes the file sorting to fail, you can modify the task configuration of TiCDC to filter out this table and use other backup and restore tools (such as [BR](/br/backup-and-restore-tool.md)) to restore the table before you resume replicating the table.
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: ticdc/troubleshoot-ticdc.md (hunk 1 above)
Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:65]: Applied in https://github.com/pingcap/docs/pull/3881#discussion_r486778489.

Comment (amyangfei) [reply ticdc/troubleshoot-ticdc.md:65]: looks good

**Review by amyangfei — COMMENTED** [review]
LGTM

**Comment by TomShawn** [comment]
@amyangfei PTAL again, thanks!

**Pull request merged**